client/src/actions/auth.ts

```diff
@@ -0,0 +1,132 @@
+import axios from "axios";
+
+import {
+  REGISTER_SUCCESS,
+  REGISTER_FAILURE,
+  LOGIN_SUCCESS,
+  LOGIN_FAILURE,
+  LOG_OUT,
+  AUTH_ERROR,
+  USER_LOADED
+} from "./types";
+
+import setAuthToken from "../utils/setAuthToken";
+
+const config = {
+  headers: {
+    "Content-Type": "application/json",
+  },
+};
+
+export const loadUser = () => async (dispatch: any): Promise<void> => {
+  if (localStorage.token) {
+    setAuthToken(localStorage.token);
+  }
+
+  try {
+    const res = await axios.get("http://localhost:5000/api/auth/user");
```

Comment:
Create base url. Actually you can try to create axios http service for requests.

Reply:
done